base/sparse/cholmod.jl

```diff
@@ -1603,6 +1603,9 @@ Ac_ldiv_B(L::FactorComponent, B::RowVector) = ctranspose(L)\B # ambiguity
 # linalg/factorizations.jl
 (\){T<:VTypes}(L::Factor{T}, b::StridedVector) = Vector(L\convert(Dense{T}, b))
 (\){T<:VTypes}(L::Factor{T}, B::StridedMatrix) = Matrix(L\convert(Dense{T}, B))
+# nix ambiguities with (\){T<:BlasReal}(F::Factorization{T}, V::VecOrMat{Complex{T}})
+(\){T<:Float64}(L::Factor{T}, B::Vector{Complex{T}}) = Vector(L\convert(Dense{T}, B))
```

Comment:
Is this to include `Union{}`? If that is the case then I really think we should fix the ambiguity check instead of adding methods for instances that cannot be constructed.